tools/commit-checker/README.md

```diff
@@ -39,4 +41,23 @@ Run the tool and save the result into REPORT.md; set fixVersion to the upcoming
         --rev-range "release-63-1..upstream/maint/maint-64"
         > REPORT.md
```

Comment:
I wonder if it might be worth adding a line or two saying something like:
Note: If the maintenance branch has not yet been created then use the master branch in the revision.
Ex: "release-64-2..upstream/master"
(Also this assumes that the unicode-org repo is named upstream).

Reply:
Done